src/main/java/by/teachmeskills/page/LoginPage.java

```diff
@@ -0,0 +1,77 @@
+package by.teachmeskills.page;
+import by.teachmeskills.util.PropertiesLoader;
+import lombok.extern.log4j.Log4j2;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.Properties;
+public class LoginPage extends BasePage {
+
+    private static final By USER = By.id("login");
+    private static final By PASSWORD = By.id("password");
+    private static final By LOGIN_PAGE_IS_OPEN_TEXT = By.xpath("//h1[text()=' Login ']");
+    private static final By LOGIN_FAILED = By.xpath("//div[text()=' Login failed ']");
+    private static final By MANDATORY_FIELD_HINT = By.xpath("//div[text()=' Mandatory field ']");
+    public static final By MAIN_PAIGE_LOGIN_BUTTON = By.xpath("//a[text()='Login']");
+    private static final By LOGIN_BUTTON = By.xpath("//div[text()=' Login ']");
+    public static final String LOGIN_FAILED_TEXT = "Login failed";
+    public static final String MANDATORY_FIELD_TEXT = "Mandatory field";
+
+    public List<WebElement> displayedMandatoryFieldHints = driver.findElements(MANDATORY_FIELD_HINT);
```

Comment:
this code should be moved in methods -  driver.findElements(MANDATORY_FIELD_HINT);
you can leave vars declaration here, but driver.findElements(...); should be in corresponding methods